common/api-review/auth.api.md

```diff
@@ -750,10 +750,40 @@ export function signOut(auth: Auth): Promise<void>;
 export interface TotpMultiFactorAssertion extends MultiFactorAssertion {
 }
 
+// @public
+export class TotpMultiFactorGenerator {
+    static assertionForEnrollment(secret: TotpSecret, oneTimePassword: string): TotpMultiFactorAssertion;
+    static assertionForSignIn(enrollmentId: string, oneTimePassword: string): TotpMultiFactorAssertion;
+    // Warning: (ae-forgotten-export) The symbol "FactorId" needs to be exported by the entry point index.d.ts
+    static FACTOR_ID: FactorId_2;
+    static generateSecret(session: MultiFactorSession): Promise<TotpSecret>;
+}
+
 // @public
 export interface TotpMultiFactorInfo extends MultiFactorInfo {
 }
 
+// @public
+export class TotpSecret {
+    // (undocumented)
```

Comment:
I'm not sure why this got included now (maybe because the files were added to the index?) but we should make sure anything that's undocumented is marked internal, or is documented

Reply:
These are public fields and were documented in the constructor - https://github.com/firebase/firebase-js-sdk/blob/mfa-totp/packages/auth/src/mfa/assertions/totp.ts#L175-L178

Looks like I would need to declare each member with a doc, rather than implicitly define them in the constructor. 

Fixed it.